.github/workflows/ci.yml

```diff
@@ -73,7 +81,9 @@ jobs:
         ./run-qa-checks
     - name: Tests
       run: |
-        coverage run --source=openwisp_controller runtests.py
+        coverage run --parallel-mode --source=openwisp_controller runtests.py
+        POSTGRESQL=1 coverage run --parallel-mode --source=openwisp_controller runtests.py
+        coverage combine
```

Comment:
Nice :+1:!
Looks like `--parallel-mode` is something we can start using in other repos too right?

Reply:
Please note that the Django tests are not running with `--parallel`. The `--parallel-mode` flag instructs coverage.py to write coverage to distinct files. A quick web search yielded this https://stackoverflow.com/a/71425897/7911552. We should definitely try this to improve CI build time.